app/src/main/java/ai/brokk/analyzer/usages/FuzzyUsageFinder.java

```diff
@@ -269,7 +269,7 @@ public FuzzyResult findUsages(String fqName, int maxFiles, int maxUsages) {
             logger.debug("Project/analyzer empty; returning empty Success for fqName={}", fqName);
             return new FuzzyResult.Success(Set.of());
         }
-        var maybeCodeUnit = analyzer.getDefinition(fqName);
+        var maybeCodeUnit = analyzer.getDefinitions(fqName).stream().findFirst();
```

Comment:
`findAny` has some performance benefits for ordered streams, and since this operation is just checking for existence, we can use it instead
```suggestion
        var maybeCodeUnit = analyzer.getDefinitions(fqName).stream().findAny();
```